pkg/transformer/openshift/openshift.go

```diff
@@ -622,23 +624,25 @@ func (o *OpenShift) Undeploy(komposeObject kobject.KomposeObject, opt kobject.Co
 			//delete pods
 			pod, err := kclient.Pods(namespace).List(options)
 			if err != nil {
-				return err
+				ErrorList = append(ErrorList, err)
 			}
 			for _, l := range pod.Items {
 				if reflect.DeepEqual(l.Labels, komposeLabel) {
 					rpPod, err := kubectl.ReaperFor(api.Kind("Pod"), kclient)
 					if err != nil {
-						return err
+						ErrorList = append(ErrorList, err)
 					}
+
 					//FIXME: gracePeriod is nil
 					err = rpPod.Stop(namespace, t.Name, TIMEOUT*time.Second, nil)
 					if err != nil {
-						return err
+						ErrorList = append(ErrorList, err)
 					}
 					log.Infof("Successfully deleted Pod: %s", t.Name)
+
 				}
 			}
 		}
 	}
-	return nil
+	return ErrorList
```

Comment:
> This seems a bit unconventional for Go, why an ErrorList? We should be returning an error when it actually happens and not continue throughout the code, should this be nil? @kadel

I don't think that this is against Go conventions. It quite common to aggregate errors into slice. If you look at Kubernetes code base you can see it there. Only difference is that they are doing it little bit smarter. They don't return slice of errors, but `Aggregate`. Aggregate is special type that implements error interface and can contain multiple errors. (see https://github.com/kubernetes/kubernetes/blob/55bee3ad21f025b1416a4e1f10de753f484b66d3/staging/src/k8s.io/apimachinery/pkg/util/errors/errors.go#L36).
I don't think we need to do that like that right now, just returning `[]error` should be enough.

Something went wrong, even if we tried to continue but we need to inform user that something wasn't deleted.

This is simplification, because ideally deletion should happen in parallel. Every delete should have run its own goroutine. But I don't think we should worry about that right now. Doing it serially on-by-one is ok for now.